### DISCUSSION TITLE
posts/2023/06/hugo-tailwind-peace-at-last-maybe/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#29] [Announcements]
# posts/2023/06/hugo-tailwind-peace-at-last-maybe/

A new enhancement to the most powerful SSG enables it to work smoothly with the most popular CSS framework — at least, until the latter’s next major release.

https://www.brycewray.com/posts/2023/06/hugo-tailwind-peace-at-last-maybe/

**Comment by gursuj** [comment]
If anyone stumbles across this, bep himself made a starter theme that works perfectly with tailwind.
https://github.com/bep/hugo-starter-tailwind-basic

You might want to comment out the `disableKinds` in the `hugo.toml` file

**Comment by brycewray** [comment]
@dinesh-58 Yes, his starter is mentioned in (and linked from) the linked Hugo 0.112.0 release notes. Definitely worth trying.

As for the `hugo.toml` file: presuming you’re talking about the one in my site repo, it’s a long story why I have it that way, but I do have reasons. :)

**Reply by gursuj** [reply]
Oh I was referring to the toml file in the theme repo.
